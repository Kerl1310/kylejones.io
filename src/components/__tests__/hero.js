import React from 'react';
import renderer from 'react-test-renderer';

import Hero from '../hero';

describe('Hero', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Hero heroImg="/my-image.jpeg" title="my-title" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
