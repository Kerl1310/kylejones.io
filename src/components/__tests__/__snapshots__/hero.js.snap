// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div
  className="sc-bdVaJa gjdwiW sc-bxivhb cCNszn"
>
  <div
    className="sc-htpNat brSZyA"
  >
    <h1
      className="sc-bwzfXH fUSFxt"
    >
      my-title
    </h1>
  </div>
</div>
`;
